README.md

```diff
@@ -78,7 +78,7 @@ Please read [METRICS.md](METRICS.md) for the setup steps.
 
 
 ```bash
-docker compose build
+docker compose pull
 docker compose up
```

Comment:
```suggestion
docker compose --pull up
```

I think this should work (see [documentation](https://docs.docker.com/reference/cli/docker/compose/up/))